### PR TITLE
TST: Fix flaky test

### PR DESCRIPTION
**Pull request by jbrockmendel**
Addresses one of the checkboxes in #26823

### CODE REVIEW

**Review by jreback — CHANGES_REQUESTED** [review]
can u make these fixtures which will just fix this in a standard way

**Comment by jbrockmendel** [comment]
fixturized

File: pandas/tests/resample/test_resample_api.py [hunk 1]
```diff
@@ -14,10 +14,15 @@
                  end=datetime(2005, 1, 10), freq='Min')
 
 test_series = Series(np.random.rand(len(dti)), dti)
-test_frame = DataFrame(
+_test_frame = DataFrame(
```
Comment (jreback) [inline pandas/tests/resample/test_resample_api.py]: can u just inline this in the fixture?

can u fixturize the series as well

not so sure these are great names either

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/resample/test_resample_api.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/resample/test_resample_api.py]: Fixturing this kind of thing is like pulling my own teeth.  Closing.

Comment (jreback) [reply pandas/tests/resample/test_resample_api.py]: ok i can’t imagine it taking very long

**Comment by jorisvandenbossche** [comment]
Fix itself seems good, so merging

**Pull request merged**